apps/server/src/services/TimerService.ts

```diff
@@ -43,6 +45,12 @@ export class TimerService {
     this._updateInterval = timerConfig?.updateInterval ?? 1000;
   }
 
+  
+  public get paused() : number | null {
+    return this.pausedAt;
+  }
+  
+
```

Comment:
I am unsure I understand what this achieves

Reply:
was needed when eventStore was responsible for saving
forgot to delete it.